src/Raven.Server/Documents/DatabaseOperations.cs

```diff
@@ -268,9 +268,14 @@ public enum PendingOperationType
 
             DatabaseExport,
 
-            IndexCompact
+            IndexCompact,
 
+            DeleteByCollection,
+
+            PatchByCollection
```

Comment:
Inconsistent with UpdateByIndex. Should be UpdateByCollection? Or the other way...